ex01.ex

```diff
@@ -86,10 +86,9 @@ defmodule Ex01 do
   # function. The examples below will make this clearer :)               #
   ########################################################################
 
-  create_adder = your_anonymous_function(1)
-
-  add_2  = create_adder.(2)
-  add_99 = create_adder.(99)
+create_adder = fn a -> a end
```

Comment:
I am still not able to get the answer for this. I followed the given code structure.

create_adder = your_anonymous_function(1)

  add_2  = create_adder.(2)
  add_99 = create_adder.(99)

  assert add_2.(3)  == 5
  assert add_99.(3) == 102

end